Sources/SQLKit/Builders/SQLAlterTableBuilder.swift

```diff
@@ -1,290 +1,158 @@
+/// Builds ``SQLAlterTable`` queries.
 public final class SQLAlterTableBuilder: SQLQueryBuilder {
-    /// `SQLAlterTable` query being built.
+    /// ``SQLAlterTable`` query being built.
     public var alterTable: SQLAlterTable
 
-    public var database: SQLDatabase
+    /// See ``SQLQueryBuilder/database``.
+    public var database: any SQLDatabase
 
-    public var query: SQLExpression {
-        return self.alterTable
+    /// See ``SQLQueryBuilder/query``.
+    @inlinable
+    public var query: any SQLExpression {
+        self.alterTable
     }
 
-    public var columns: [SQLExpression] {
-        get { return alterTable.addColumns }
-        set { alterTable.addColumns = newValue }
+    /// The set of column alteration expressions.
+    @inlinable
+    public var columns: [any SQLExpression] {
+        get { self.alterTable.addColumns }
+        set { self.alterTable.addColumns = newValue }
     }
 
-    /// Creates a new `SQLAlterTableBuilder`.
-    public init(_ alterTable: SQLAlterTable, on database: SQLDatabase) {
+    /// Create a new ``SQLAlterTableBuilder``.
+    @inlinable
+    public init(_ alterTable: SQLAlterTable, on database: any SQLDatabase) {
         self.alterTable = alterTable
         self.database = database
     }
     
-    @discardableResult
     /// Rename the table.
-    /// - Parameter newName: The new name to apply to the table
```

Comment:
What's with removing the parameters from the docs? These would be good for DocC right?

Reply:
In basically every case, they're pointless. "The name is the name" etc.; it's just absurd.

Reply:
Ok let's leave it, we can always backfill if necessary